be/src/exec/sorting/merge_path.cpp

```diff
@@ -1133,6 +1135,13 @@ void MergePathCascadeMerger::_init_late_materialization() {
             metrics.profile->add_info_string("LateMaterialization", _late_materialization ? "True" : "False");
         });
     });
+    if (_late_materialization_mode == TLateMaterializeMode::ALWAYS) {
```

Comment:
should we keep the same value of `spill` like `auto` and `force`?

Reply:
It seems unnecessary.